pipeline/mnist/mnist_datasets_treatment.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+import struct as st
+import math
+
+training_filenames = {'images': mnist_train_images,
+                      'labels': mnist_train_labels}
+test_filenames = {'images': mnist_test_images,
+                  'labels': mnist_test_labels}
+
+data_types = {
```

Comment:
please, change the data_types to DATA_TYPES, according PEP8 constants variables are defined with caps lock.

Reply:
you can use the refactor-> rename feature from pycharm, this feature change all occurrences of this variable automatically without need of change manually each occurrence.